data/test/vtgate/select_cases.txt

```diff
@@ -460,6 +460,31 @@
     "FieldQuery": "select * from user where 1 != 1"
   }
 }
+# sharded limit offset
+"select user_id from music order by user_id limit 10, 20"
+{
+  "Original": "select user_id from music order by user_id limit 10, 20",
+  "Instructions": {
+    "Opcode": "Limit",
+    "Count": 20,
+    "Offset": 10,
+    "Input": {
+      "Opcode": "SelectScatter",
+      "Keyspace": {
+        "Name": "user",
+        "Sharded": true
+      },
+      "Query": "select user_id from music order by user_id asc limit 20",
```

Comment:
This should be `limit 30`. However, given that 10 and 20 could be bind vars, the limit cannot be pre-computed at planbuilder time.

So, one option is to generate something `limit :__upper_limit`, and compute the value for this bind var as `count+limit` at execution time.